lib/platform/config.go

```diff
@@ -232,9 +232,11 @@ func (c *Config) GetPGConnInfoURI(user string, opts ...ConnInfoOpts) (*PGConnInf
 					user, c.GetPostgresql().GetIp(), c.GetPostgresql().GetCfg().GetPort(), strings.Join(opts, "&"))
```

Comment:
Just from pattern matching, you have added timezone awareness in both the format string and the env in L258-267. However, here you have added it only to the env (L328), not the format string (L229). Is there a reason for the difference?

Reply:
There's no difference. I've added it to `opts` in L196, and it's put into the format string using `strings.Join(opts, "&")`.

Reply:
ℹ️ Rolled back the URI change, it's only supported by the golang lib, not libpq. (pg_dump etc)